### ISSUE TITLE
Scroll to Top

### ISSUE THREAD

**Issue by archanasingh11** [Sanskriti1102/Techtalksintelligenz#9]
**Description:**

The TechTalksIntelligenz website currently lacks a **"Scroll to Top"** button, which is a key feature for enhancing user experience, especially on pages with extensive content. Without this button, users have to manually scroll back to the top of the page, which can be inconvenient and time-consuming.

Implementing a **"Scroll to Top"** button would:
- Improve the website's accessibility and usability.
- Provide a smoother navigation experience, particularly on long articles or blog posts.
- Align the user interface with modern design practices.

I recommend adding a floating "Scroll to Top" button that becomes visible as users scroll down the page.

Please @Sanskriti1102 assign me this task with labels "GSSOC-Ext", "Hacktoberfest-accepted" and"level1".

**Comment by Sanskriti1102** [comment]
@archanasingh11  since this project isn't under hacktoberfest so no label of that will be added

**Comment by archanasingh11** [comment]
Hey @Sanskriti1102 could you please add the label "level2" for gssoc-ext'24

**Comment by akshatjaiin** [comment]
can i work with her under gssoc or hackerfest

**Comment by Krishi1211** [comment]
@archanasingh11 are u still working on this?

**Comment by archanasingh11** [comment]
yes im still working on this and i will complete this by tonight

**Comment by Sanskriti1102** [comment]
@archanasingh11 may we know when u will be submitting the above ?

**Comment by archanasingh11** [comment]
today

**Comment by akshatjaiin** [comment]
if u need any help feel free to assign that to me

**Comment by Sanskriti1102** [comment]
@archanasingh11 since you aren't responding and despite asking for the updations, we have assigned this issue to @akshatjaiin

**Comment by IkkiOcean** [comment]
@Sanskriti1102 

assign this to me please. I can implement it and PR in one day.

**Comment by Sanskriti1102** [comment]
@IkkiOcean assigned

**Comment by Sanskriti1102** [comment]
@akshatjaiin are you still working?

**Comment by IkkiOcean** [comment]
@Sanskriti1102 thanks for assigning. I will PR tonight

**Comment by akshatjaiin** [comment]
i forget about this
